zmon-controller-ui/views/alertDetails.html

```diff
@@ -156,7 +156,7 @@
                                     </div>
                                 </td>
                                 <td>
-                                    <div class="ellipsis" expand>{{entityInstance.result.value}}
+                                    <div class="ellipsis" alert-value-modal name="entityInstance.entity" value="entityInstance.result.value">{{entityInstance.result.value}}
```

Comment:
it should be ok, since the modal controller and the $watch for 'filter' will only be created after the user click, destroyed on close.

In other words there should only be one modal controller and one filter $watch at a given time (I tested this).